RecoTracker/MkFit/plugins/MkFitHitConverter.cc

```diff
@@ -91,6 +97,9 @@ MkFitHitConverter::MkFitHitConverter(edm::ParameterSet const& iConfig)
       mkFitGeomToken_{esConsumes<MkFitGeometry, TrackerRecoGeometryRecord>()},
       wrapperPutToken_{produces<MkFitHitWrapper>()},
       clusterIndexPutToken_{produces<MkFitClusterIndexToHit>()},
+      //qualityToken_{esConsumes<edm::Transition::BeginRun>()},
+      qualityToken_{esConsumes<edm::Transition::Event>()},
+      geomToken_{esConsumes<edm::Transition::Event>()},
```

Comment:
The `edm::Transition::Event` is the default and therefore redundant
```suggestion
      qualityToken_{esConsumes()},
      geomToken_{esConsumes()},
```

Reply:
I left `edm::Transition::Event` (and the commented out BeginRun version) since I think in principle this is not an event-by-event record and at some point we may want to check it less often (but not sure what the right transition would be... is run too coarse?).
But I am not opposed to the suggestion if it is preferred.

Reply:
The transition parameter to `esConsumes()` has nothing to do with the IOV of the Record, but with the transition function where the products are used. The recommended pattern is to get the products in the transitions where they are needed, the overhead from `EventSetup::getData()` / `EventSetup::getHandle()` are tiny and then one doesn't have to worry about the IOV structure of the Record.

Reply:
Ok, I guess this exposes my ignorance about what transitions are. Anyways, I will accept the suggestion